pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/OsteoMRFv2.java

```diff
@@ -47,18 +48,15 @@
 import org.jdbi.v3.sqlobject.customizer.Bind;
 import org.jdbi.v3.sqlobject.statement.SqlQuery;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * Task to make additional edits as part of the "Brain Tumor Project" rename.
```

Comment:
Looks like this study-builder task was copied from Brain and comments not updated.
Dmitri , not your problem though.